### ISSUE TITLE
PM UI Icons aren't inverting colors properly

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#10017]
Icons in PMUI cannot properly use the Image Service since the ImageBackgroundColor property is not being set.
This causes icons not to invert their colors, and not look appropriate for the VS Theme.

Additionally, the Refresh icon will pixelate since it's not using the service (reported in https://github.com/NuGet/Home/issues/5937)
